### DISCUSSION TITLE
s/simple-reliable-and-free-home-photo-backup-solution

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#70] [Announcements]
# s/simple-reliable-and-free-home-photo-backup-solution

照片是记忆的载体，它记录着我们的成长、旅行、欢笑与感动，是生命中最宝贵的财富之一，我们都应该珍惜

https://blog.ops-coffee.cn/s/simple-reliable-and-free-home-photo-backup-solution

**Comment by thetbw** [comment]
检索的话推荐个immich,像是自托管的google相册，有时间轴本地人脸识别之类功能，有客户端网页版和cli命令行，指定外部库时可以自动扫描新增然后索引，手机端也能自动备份。

**Reply by ops-coffee** [reply]
感谢推荐

**Comment by SerendipityCoisini** [comment]
不开会员嘛？

**Reply by ops-coffee** [reply]
确实没必要

**Reply by SerendipityCoisini** [reply]
那需要下载的时候那，百度那网速，还是说他只是浏览，需要下载是再开会员

**Reply by ops-coffee** [reply]
本地不是存了一份嘛，大量浏览的时候就本地浏览，偶尔查个几张慢点影响也不大，目前阿里会快一点，临时查看优先阿里查

**Comment by yyt6801** [comment]
直接阿里云盘备份+百度云盘备份，默认的都是按照时间分组的，和up这种分组类似，不折腾省事儿

**Reply by ops-coffee** [reply]
默认分组不太好，还是自己整理下

**Comment by yisier** [comment]
老哥，博客能加个RSS吗

**Reply by ops-coffee** [reply]
有RSS呀，首页侧边栏多么明显 https://blog.ops-coffee.cn/feed.xml

**Comment by realSinged** [comment]
备份在百度云之类的地方，不怕数据隐私的问题吗

**Comment by strongbugman** [comment]
实话说，不简单也不可靠hhh

**Reply by ops-coffee** [reply]
不要YY

**Comment by ttkanni** [comment]
手机照片备份这个问题，之前折腾的时候也遇到了，分享下自己的折腾史吧：
1，最先考虑的就是各种云盘+本地，成本低、方便，但云盘是“流氓”还是“君子”，完全没法赌，明确肯定百度网盘会读取照片定期给你推“回忆杀”，OneDrive也有类似行为。
2，其次就是IPHONE+ICLOUD，家里水果设备多的话，很好的选择，Mac + IPHONE + Apple TV，通过ICLOUD无缝衔接，唯一的缺憾就是ICLOUD的上传下载速度是个迷，有时候很快，有时候慢。
3，最后就是中年男人的爱好，NAS 黑群 + 群晖相册/immich ，稳定，在家的时候也可以线上浏览，唯一缺点--成本略高，当然要是有其他需求，比如监控+电影，整体TOC还是值得的。公网访问就要DDNS了，折腾点但也能用，尤其是IPV6之后。
